examples/convert_mri_and_trans.py

```diff
@@ -0,0 +1,158 @@
+output_path = op.abspath(op.join(data_path, '..', 'MNE-sample-data-bids'))
+if op.exists(output_path):
+    sh.rmtree(output_path)
+raw = mne.io.read_raw_fif(raw_fname)
+sub = '01'
+ses = '01'
+task = 'audiovisual'
+run = '01'
+bids_basename = make_bids_basename(subject=sub, session=ses, task=task,
+                                   run=run)
+write_raw_bids(raw, bids_basename, output_path, events_data=events_data,
+               event_id=event_id, overwrite=True)
+
+# Print the directory tree
+print_dir_tree(output_path)
+
+###############################################################################
+# Now let's assume that we have also collected some T1 weighted MRI data for
+# our subject. And furthermore, that we have already aligned our coordinate
+# frames (using e.g., the `coregistration GUI`_) and obtained a transformation
+# matrix :code:`trans`.
+
+# Get the path to our MRI scan
+t1_mgh_fname = op.join(data_path, 'subjects', 'sample', 'mri', 'T1.mgz')
+
+# Load the transformation matrix and show what it looks like
+trans_fname = op.join(data_path, 'MEG', 'sample',
+                      'sample_audvis_raw-trans.fif')
+trans = mne.read_trans(trans_fname)
+print(trans)
+
+###############################################################################
+# We can save the MRI to our existing BIDS directory and at the same time
+# create a JSON sidecar file that contains metadata, we will later use to
+# retrieve our transformation matrix :code:`trans`.
+
+# We use the write_anat function
+write_anat(bids_root=output_path,  # point to the BIDS dir we wrote earlier
+           subject=sub,
+           t1w=t1_mgh_fname,  # path to the MRI scan
+           session=ses,
+           raw=raw,  # the raw MEG data file connected to the MRI
+           trans=trans,  # our transformation matrix
+           verbose=True  # this will print out the sidecar file
+           )
+
+# Let's have another look at our BIDS directory
+print_dir_tree(output_path)
+
+###############################################################################
+# Our BIDS dataset is now ready to be shared. We can easily estimate the
+# transformation matrix using ``MNE-BIDS`` and the BIDS dataset.
+
+bids_fname = bids_basename + '_meg.fif'
+
+# reproduce our trans
+estim_trans = get_head_mri_trans(bids_fname=bids_fname,  # name of the MEG file
+                                 bids_root=output_path  # root of our BIDS dir
+                                 )
+
+###############################################################################
+# Finally, let's use the T1 weighted MRI image and plot the anatomical
+# landmarks Nasion, LPA, and RPA (=left and right preauricular points) onto
+# the brain image. For that, we can extract the location of Nasion, LPA, and
+# RPA from the MEG file, apply our transformation matrix :code:`trans`, and
+# plot the results.
+
+# Get Landmarks from MEG file
+coords_dict = _extract_landmarks(raw.info['dig'])
```

Comment:
don't expose a private function. Use dig[0], dig[1] or maybe read the json function written
by write_anat ?